src/export/smtlib.ml

```diff
@@ -1,71 +1,86 @@
 (* This module contains the pretty-printer functions for smtlib *)
 
 open Dolmen_std
-
+open Pretty
 open Term
```

Comment:
I'm really, really, really not a fan of using a lot of opens. `Dolmen_std` is fine to open, but `Pretty` and `Term` should not be opened.